### PR TITLE
Bugfix jumping sidebar

### PR DESCRIPTION
**Pull request by Blackbaud-BrandonJones**
Resolves both:
issue https://github.com/blackbaud/stache2/issues/424
and
issue https://github.com/blackbaud/stache2/issues/446

### CODE REVIEW

File: src/app/public/src/modules/wrapper/wrapper.component.scss [hunk 1]
```diff
@@ -1,5 +1,15 @@
 @import '../../styles/_variables';
 
+/*
+* We add this overflow-y so that when switching between screens with and without a vertical
+* scroll bar the page doesn't "jump" 16px back and forth.
+* https://stackoverflow.com/a/7607206
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/src/modules/wrapper/wrapper.component.scss]: "Worth mentioning that this permanently shows a vertical scrollbar. May not always be acceptable." :(

**Comment by Blackbaud-SteveBrush** [comment]
I'm struggling to see the importance of fixing #424? It seems that that is just how browser pages work and I wouldn't want to mess with overflow on the html tag...

**Comment by Blackbaud-LindseyRix** [comment]
The developer who reported it is Ben Lambert. I take it seriously the requests he makes and would appreciate it if we fix this for him.

**Comment by Blackbaud-BrandonJones** [comment]
This is being resolved with the reformatting of the sidebar layout.

**Pull request closed without merging**